.github/workflows/build-package.yml

```diff
@@ -20,15 +20,15 @@ jobs:
 
       - name: Build with Maven
         env:
-          GITHUB_TOKEN: ${{ secrets.PAT_READ_WRITE }}
+          GITHUB_TOKEN: ${{ secrets.TOKEN }}
```

Comment:
PAT_READ_WRITE is the token name.

Personal Access Token for reading the repo and writing the package